gitignore/styles.css

```diff
@@ -1,4 +1 @@
-h1 {
-  color: blue;
-  font-style: italic;
-}
\ No newline at end of file
+h1 { color: blue;}
```

Comment:
**OPTIONAL** - Please ensure this file is in your root directory and not inside the the gitignore folder.